### PR TITLE
auto cases  OCP-33850 OCP-33848

### PR DESCRIPTION
**Pull request by zhaozhanqi**
@anuragthehatter @huiran0826 @weliang1 @rbbratta  PTAL, thanks

http://pastebin.test.redhat.com/899756
http://pastebin.test.redhat.com/899757
http://pastebin.test.redhat.com/899808

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
@anuragthehatter @huiran0826 @weliang1 @rbbratta PTAL,thanks

File: features/networking/service.feature [hunk 1]
```diff
@@ -479,3 +479,56 @@ Feature: Service related networking scenarios
...
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-33848
+  @admin
+  @destructive
+  Scenario: User can expand the nodePort range by patch the serviceNodePortRange in network
+    Given I have a project
+    And evaluation of `rand(32676..33000)` is stored in the :port clipboard
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"serviceNodePortRange": "30000-33000"}} |
```
Comment (liangxia) [inline features/networking/service.feature:491]: We need clean-up for this change to restore the cluster.

File: features/networking/service.feature [hunk 2]
```diff
@@ -479,3 +479,56 @@ Feature: Service related networking scenarios
...
+    When I obtain test data file "networking/nodeport_service.json"
+    And I wait up to 120 seconds for the steps to pass:
+    """    
+    When I run oc create over "nodeport_service.json" replacing paths:
+      | ["items"][1]["spec"]["ports"][0]["nodePort"] | <%= cb.port %> |
+    Then the step should succeed
+    """
+    Given the pod named "hello-pod" becomes ready
+    And evaluation of `pod('hello-pod').node_ip(user: user)` is stored in the :hostip clipboard
```
Comment (liangxia) [inline features/networking/service.feature:500]: You can use `pod.node_ip` for short.

File: features/networking/service.feature [hunk 3]
```diff
@@ -479,3 +479,56 @@ Feature: Service related networking scenarios
...
+      | name=test-pods |
+    When I execute on the pod:
+      | curl | <%= cb.hostip %>:<%= cb.port %> |
+    Then the step should succeed
+    And the output should contain:
+      | Hello OpenShift! |
+    When I run the :delete client command with:
+      | object_type | service |
+      | object_name_or_id | hello-pod |
+    Then I wait for the resource "service" named "hello-pod" to disappear
```
Comment (liangxia) [inline features/networking/service.feature:516]: Above 4 lines can be shorten as `Given I ensure "xxx" service is deleted`

File: features/networking/service.feature (hunk 1 above)
Comment (zhaozhanqi) [reply features/networking/service.feature:491]: @liangxia  User cannot decrease the nodePort range in post action, So once the serviceNodePortRange is expand, it cannot be restored to origin(30000-32676).

File: features/networking/service.feature (hunk 2 above)
Comment (zhaozhanqi) [reply features/networking/service.feature]: updated. thanks

File: features/networking/service.feature (hunk 3 above)
Comment (zhaozhanqi) [reply features/networking/service.feature]: updated. thanks

File: features/networking/service.feature (hunk 1 above)
Comment (liangxia) [reply features/networking/service.feature:491]: Ack

**Pull request merged**